src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx

```diff
@@ -82,6 +84,10 @@ export const MeetTheSpecialists: React.FC = () => {
     })
   }
 
+  const clickContactSpecialist = email => {
+    router.push(`/sell/inquiry/${email}`)
+    trackContactTheSpecialistClick()
```

Comment:
I think you want to trigger the tracking before pushing a screen onto the stack

Reply:
i've replaced it with router link